packages/erxes-ui/src/components/CollapseSidebar.tsx

```
@@ -0,0 +1,81 @@
+};
+
+const CollapsibleDiv = ({ children, title, onToggle, isCollapsed }: Props) => {
+  const [collapsed, setCollapsed] = useState(isCollapsed || false);
+
+  useEffect(() => {
+    isCollapsed && setCollapsed(isCollapsed);
+  }, [isCollapsed]);
+
+  const toggleCollapse = () => {
+    setCollapsed(!collapsed);
+    onToggle && onToggle(false);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)